### PR TITLE
Update BERT convergence test baseline.

### PR DESCRIPTION
**Pull request by codemzs**
BERT-LOSS was changed recently where the loss function was replaced with SoftmaxCrossEntropyLoss which caused subtle changes in the convergence test output. I have verified that results are stable within 0.001 tolerance range by running it 5 times and also ran performance test with 10K steps, below are the graphs. 

image

image

### CODE REVIEW

**Comment by ytaous** [comment]
will be nice to run convergence test for 10000 steps (~ couple hours)

**Comment by codemzs** [comment]
@ytaous I have started the perf run, will check the result in the morning.

**Comment by codemzs** [comment]
@ytaous 

image
